### DISCUSSION TITLE
About deployment of V2 via Openshift

### DISCUSSION THREAD

**Discussion by CicoZhang** [alfio-event/alf.io#1080] [Deployment]
Hi, everyone:

I am considering deploying an instance via OpenShift (everything is totally new to me), [https://alf.io/docs/getting-started/deployment/openshift/](https://alf.io/docs/getting-started/deployment/openshift/), but it says

> This tutorial refers to a version of Alf.io (1.x) which is now EOL. Please help us updating it!

Should I follow this document? Is there any significant change that I should be aware of?

Thank you, have a peaceful day.

**Comment by cbellone** [comment]
Hi, 

The Openshift installation guide was contributed by a community member, and it hasn't been tested/validated with the new v2 builds.
It would be great if you could follow the process with the updated resources and validate if it still works on v2.

The descriptor linked in the guide is old and it is likely to contain old/deprecated/removed Openshift resources. 
So the main task would be to validate/fix it.  

Once we have a working configuration we can update the tutorial

Thank you for your help!

**Reply by CicoZhang** [reply]
Good to know. I will try to deploy it by following that document, if anything does not work, I will let you know. Have a peaceful day.
